ui/components/app/transaction-list/transaction-list.component.js

```diff
@@ -123,23 +123,36 @@ export default function TransactionList({
             <div className="transaction-list__header">
               {`${t('queue')} (${pendingTransactions.length})`}
             </div>
-            {pendingTransactions.map((transactionGroup, index) =>
-              transactionGroup.initialTransaction.transactionType ===
-              TransactionType.smart ? (
-                <SmartTransactionListItem
-                  isEarliestNonce={index === 0}
-                  smartTransaction={transactionGroup.initialTransaction}
-                  transactionGroup={transactionGroup}
-                  key={`${transactionGroup.nonce}:${index}`}
-                />
-              ) : (
-                <TransactionListItem
-                  isEarliestNonce={index === 0}
-                  transactionGroup={transactionGroup}
-                  key={`${transactionGroup.nonce}:${index}`}
-                />
-              ),
-            )}
+            {pendingTransactions
```

Comment:
Can we add some snapshot tests for this condition?

Reply:
It wouldn't be helpful in this case. Please let me know if I'm wrong. We are just sorting transactions when it is MMI, just it.

Reply:
If you want to cover, you can assert the items of txn one by one to see if the order is sorted. but it's optional ~ :)